.drone.yml

```diff
@@ -63,6 +63,8 @@ steps:
       - go build -o release/windows/amd64/drone-ecr.exe ./cmd/drone-ecr
       - go build -o release/windows/amd64/drone-gcr.exe ./cmd/drone-gcr
       - go build -o release/windows/amd64/drone-acr.exe ./cmd/drone-acr
+      - go build -o release/windows/amd64/drone-gcr.exe ./cmd/drone-gar
+
```

Comment:
you are missing the windows gar docker build step

Reply:
done